### PR TITLE
Multiple improvement see BSCOS-1494

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: stock_reception_screen/models/stock_move.py [hunk 1]
```diff
@@ -27,3 +28,16 @@ def action_select_move(self):
...
+        for record in self:
+            supplier_info = fields.first(
+                record.product_id.seller_ids.filtered(
+                    lambda r: r.product_code and r.product_id == record.product_id
+                )
+            )
+            if supplier_info:
+                record.vendor_code = supplier_info.product_code
+            else:
+                record.vendor_code = ""
```
Comment (mmequignon) [inline stock_reception_screen/models/stock_move.py:43]: Isn't `False` a better empty value, as it is the Odoo empty value?

File: stock_reception_screen/models/stock_reception_screen.py [hunk 2]
```diff
@@ -194,24 +201,14 @@ def _compute_picking_filtered_move_lines(self):
...
-
-    @api.depends("current_move_product_id")
-    def _compute_current_move_product_vendor_code(self):
-        for wiz in self:
-            supplier_info = fields.first(
-                wiz.current_move_product_id.seller_ids.filtered(
-                    lambda r: r.product_code
-                    and r.product_id == wiz.current_move_product_id
+                wiz.current_step_focus_field = steps[self.current_step].get(
+                    "focus_field", ""
```
Comment (mmequignon) [inline stock_reception_screen/models/stock_reception_screen.py:210]: Same here, I would have used `False` instead of an empty string.

**Review by mmequignon — APPROVED** [review]
LG !

**Comment by guewen** [comment]
the failing tests seem unrelated and I guess @sebalix 's base branch needs a rebase

**Comment by sebalix** [comment]
I rebased this PR in https://github.com/sebalix/wms/pull/7, closing this one.

**Pull request closed without merging**